models/comment.rb

```diff
@@ -108,6 +108,7 @@ def to_hash(params={})
                  .merge("votes" => votes.slice(*%w[count up_count down_count point]))
                  .merge("abuse_flaggers" => abuse_flaggers)
                  .merge("type" => "comment")
+                 .merge("child_comments_count" => Comment.where({"parent_id" => _id}).count())
```

Comment:
@symbolist 
1. makes sense; will update.
2. do you have a better approach in mind? One could be we save count in DB; that will require refactoring to maintain that count on create/delete operations and also an update for existing data objects.